### PR TITLE
Configure Gitpod

### PR DESCRIPTION
**Pull request by mikenikles**
## Overview

This PR configures Gitpod so that anyone can start a cloud-based developer environment to experience RedwoodJS with a click of a button.

[Open in Gitpod](https://gitpod.io/#https://github.com/redwoodjs/gitpod-starter/pull/9)

## Next steps

Before merging this PR, please complete the following tasks:

- [x] Connect this project to Gitpod via https://gitpod.io/new
- [x] Wait for a Gitpod prebuild to run successfully
- [x] Start a new Gitpod environment with the "Open in Gitpod" button above to verify that Prebuilds run successfully (you shouldn't have to wait for `yarn install`)

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -4,10 +4,10 @@
 # into version control.
 
 # schema.prisma defaults
-DATABASE_URL=file:./dev.db
+DATABASE_URL=postgresql://gitpod@localhost/dev
 
 # location of the test database for api service scenarios (defaults to ./.redwood/test.db if not set)
-# TEST_DATABASE_URL=file:./.redwood/test.db
+# TEST_DATABASE_URL=postgresql://gitpod@localhost/test
```
Comment (thedavidprice) [inline .env.defaults]:
```suggestion
TEST_DATABASE_URL=postgresql://gitpod@localhost/test
```

Uncomment when using Postgres

**Comment by thedavidprice** [comment]
@mikenikles This worked like a charm! Only 2 remaining questions:
1. can we remove the file `renovate.json`? My naive approach would be to delete it within the Dockerfile
2. What Gitpod button shoud we add to the Redwood Tutorial?!?

**Comment by mikenikles** [comment]
1. Done in [30ac7b1](https://github.com/redwoodjs/gitpod-starter/pull/9/commits/30ac7b1c5e11588b133c0de967bb1b3b8cd3e3de)
2. You can use one of the following:
  * Markdown: `[Open in Gitpod](https://gitpod.io/#https://github.com/redwoodjs/gitpod-starter)`
  * HTML: `<a href="https://gitpod.io/#https://github.com/redwoodjs/gitpod-starter">Open in Gitpod</a>`

Thanks for your support with all that. I can't wait to see the impact it has on the RedwoodJS community 🥰!

**Comment by thedavidprice** [comment]
Just fantastic @mikenikles 🚀 And super excited as well.

Final RC is being published on Tuesday. I'll be in touch after that!!

**Pull request merged**